packages/cozy-client/src/store/queries.js

```diff
@@ -172,7 +172,7 @@ const getQueryDocumentsChecker = query => {
 }
 
 const makeCaseInsensitiveStringSorter = attrName => item => {
-  const attrValue = item[attrName]
+  const attrValue = get(item, attrName)
   return isString(attrValue) ? attrValue.toLowerCase() : attrValue
 }
```

Comment:
I'd label this commit as a feat.

feat: Client-side query sort handles nested attributes